src/Core/src/Platform/iOS/ViewExtensions.cs

```diff
@@ -261,30 +261,56 @@ public static void UpdateBorder(this UIView platformView, IView view)
 		public static T? FindDescendantView<T>(this UIView view) where T : UIView =>
 			FindDescendantView<T>(view, (_) => true);
 
+
 		public static void UpdateBackgroundLayerFrame(this UIView view)
 		{
 			if (view == null || view.Frame.IsEmpty)
 				return;
 
 			var layer = view.Layer;
 
-			UpdateBackgroundLayerFrame(layer, view.Bounds);
+			UpdateBackgroundLayerFrame(layer, view.Bounds, view is WrapperView);
 		}
 
-		static void UpdateBackgroundLayerFrame(CALayer layer, CGRect bounds)
+		static void UpdateBackgroundLayerFrame(CALayer layer, CGRect bounds, bool getAllSubLayers)
 		{
 			if (layer == null || layer.Sublayers == null || layer.Sublayers.Length == 0)
 				return;
 
-			foreach (var sublayer in layer.Sublayers)
+			IEnumerable<CALayer> subLayers;
+
+			if (getAllSubLayers)
+				subLayers = layer.GetLayers();
+			else
+				subLayers = layer.Sublayers;
```

Comment:
This is a comment I have already said to other developers. The issue of not using the ternary operator is that the compiler is going to do: 
```csharp
IEnumerable<CALayer> subLayers = null;
```
If you enable nullability, the compiler will return an annoying warning because you are trying to assign null to a not nullable variable. Using a ternary operation fixes the case:
```suggestion
			IEnumerable<CALayer> subLayers = getAllSubLayers ? 
				layer.GetLayers() :  layer.Sublayers;
```

Reply:
Even use var if you want so that the compiler infers it.

Reply:
Good one!